seccomp/default_linux.go

```diff
@@ -646,6 +646,34 @@ func DefaultProfile() *Seccomp {
 				Arches: []string{"s390", "s390x"},
 			},
 		},
+		{
+			LinuxSyscall: specs.LinuxSyscall{
+				Names: []string{
+					"unshare",
+				},
+				Action: specs.ActAllow,
+				Args: []specs.LinuxSeccompArg{
+					{
+						Index:    0,
+						Value:    unix.CLONE_NEWNS,
+						ValueTwo: unix.CLONE_NEWNS,
+						Op:       specs.OpMaskedEqual,
+					},
+					{
+						Index:    0,
+						Value:    unix.CLONE_NEWUTS,
+						ValueTwo: unix.CLONE_NEWUTS,
+						Op:       specs.OpMaskedEqual,
+					},
+					{
+						Index:    0,
+						Value:    unix.CLONE_NEWUTS,
+						ValueTwo: unix.CLONE_NEWNS,
+						Op:       specs.OpMaskedEqual,
+					},
+				},
+			},
+		},
```

Comment:
`CLONE_NEWUSER` does not seem covered in this PR.

Anyway, it might be still scary to allow `CLONE_NEWUSER` by default, due to its several vulnerabilities in the past (CVE-2023–0386, etc.)

Reply:
> CLONE_NEWUSER does not seem covered in this PR.

Whoops, I somehow messed up PR and did CLONE_UTS twice.